### PR TITLE
python-numpy: link against libandroid-complex-math

### PR DESCRIPTION
**Pull request by licy183**

### CODE REVIEW

**Comment by xtkoba** [comment]
What is "symbol error" here? Without description we cannot confirm if this is actually fixing anything.

**Comment by licy183** [comment]
OK. I will fix the description. Error is like `dlopen failed: cannot locate symbol "clogl" referenced by "@TERMUX_PREFIX@/python3.11/site-packages/numpy/core/_multiarray_umath.cpython-311.so"`.

File: packages/python-numpy/build.sh [hunk 1]
```diff
@@ -31,7 +32,7 @@ termux_step_configure() {
 	. ${_CROSSENV_PREFIX}/bin/activate
 	build-pip install wheel
 
-	LDFLAGS+=" -lpython${_PYTHON_VERSION} -lm"
+	LDFLAGS+=" -Wl,--no-as-needed -lpython${_PYTHON_VERSION} -l:libm.a"
```
Comment (xtkoba) [inline packages/python-numpy/build.sh:35]: Is `--no-as-needed` needed here?

**Comment by xtkoba** [comment]
Can't the shared library from `libandroid-complex-math` be used instead `libm.a`?

**Comment by licy183** [comment]
> Can't the shared library from `libandroid-complex-math` be used instead `libm.a`?

Emmm... Definitely it will work. I will do some tests...

File: packages/python-numpy/build.sh (hunk 1 above)
Comment (licy183) [reply packages/python-numpy/build.sh:35]: Actually I don't think this flag is needed but missing this flag will make some `*.so`s not link against `python3.11` and some tests will fail...

Comment (licy183) [reply packages/python-numpy/build.sh:35]: Remove this flag. When testing `LDFLAGS='-lpython3.11' python -c 'import numpy; numpy.test()'` is needed.

File: packages/python-numpy/build.sh [hunk 2]
```diff
@@ -31,7 +32,7 @@ termux_step_configure() {
 	. ${_CROSSENV_PREFIX}/bin/activate
 	build-pip install wheel
 
-	LDFLAGS+=" -lpython${_PYTHON_VERSION} -lm"
+	LDFLAGS+=" -lpython${_PYTHON_VERSION} -landroid-complex-math"
```
Comment (xtkoba) [inline packages/python-numpy/build.sh:35]: `-lm` should still be necessary because `libandroid-complex-math` is not a superset of libm.

**Comment by xtkoba** [comment]
LGTM.

**Pull request merged**